### PR TITLE
Support compileStandard

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: wrapper.js [hunk 1]
```diff
@@ -26,19 +27,32 @@ function setupMethods (soljson) {
         }
       };
     };
-    var compileInternal = soljson.cwrap('compileJSONCallback', 'string', ['string', 'number', 'number']);
-    compileJSONCallback = function (input, optimize, readCallback) {
+
+    var runWithReadCallback = function (readCallback, compile) {
```
Comment (chriseth) [inline wrapper.js]: Please add a comment that this accepts optional parameters.

**Comment by axic** [comment]
@chriseth with the binary you've compiled all of the tests fail with:
```
{
  "errors": [
    {
      "component": "general",
      "formattedMessage": "Internal exception in StandardCompiler::compileInternal",
      "message": "Internal exception in StandardCompiler::compileInternal",
      "severity": "error",
      "type": "InternalCompilerError"
    }
  ]
}
```

**Comment by axic** [comment]
It is a StandardCompiler implementation shortcoming:
```
libc++abi.dylib: terminating with uncaught exception of type Json::LogicError: in Json::Value::find(key, end, found): requires objectValue or nullValue
```

**Comment by chriseth** [comment]
Can you add `lineInfo()` to anything that catches `Exception` and provides a generic error message?

**Comment by axic** [comment]
I have fixed it in solidity, just haven't pushed the PR yet. The above message is with `catch (...)` so not sure what can be printed there.

**Comment by chriseth** [comment]
@axic ah right, it is an exception from json, so we are out of luck.

**Comment by axic** [comment]
This works properly for me. The failure on travis seems to be that `semistandard` is now incompatible with node 0.10, so we need to disable 0.10.

**Comment by axic** [comment]
Depends on #94.

**Pull request merged**